lib/internal/crypto/diffiehellman.js

```diff
@@ -219,21 +219,16 @@ function encode(buffer, encoding) {
 }
 
 function getFormat(format) {
-  let f;
   if (format) {
     if (format === 'compressed')
-      f = POINT_CONVERSION_COMPRESSED;
+      return POINT_CONVERSION_COMPRESSED;
     else if (format === 'hybrid')
-      f = POINT_CONVERSION_HYBRID;
+      return POINT_CONVERSION_HYBRID;
     // Default
```

Comment:
Can you remove this comment or move to line 231 as `POINT_CONVERSION_UNCOMPRESSED` is the default?